pkg/dc2deployment/dc2deployment_test.go

```diff
@@ -0,0 +1,94 @@
+package dc2deployment
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	dcAPI "github.com/openshift/api/apps/v1"
```

Comment:
only suggestion id have is changing `dcAPI` to `apps`. looks good otherwise!

Reply:
implemented the review comments.